### ISSUE TITLE
[DetailsList][Accessibility] Inspect tool not able to find name for "select all" checkbox

### ISSUE THREAD

**Issue by ankitk94** [OfficeDev/office-ui-fabric-react#2434]
### Bug Report

- __Browser and OS versions__: Edge

Products/sites affected: VSTS

#### Describe the issue:

Provided property "ariaLabelForSelectAllCheckbox" for DetailsList, still inspect tool not able to detect name for "select all" checkbox (although it is working fine for each row checkbox). It was working for earlier versions of office fabric.
Looks like some accessibility enhancement was made inside details list regarding accesiibility. Earlier "ariaLabelForSelectAllCheckbox" was being used when "select all checkbox" was rendered as a button, now looks like this property is used only inside tooltip for "select all checkbox".

#### Expected behavior:

Should be able to provide aria-label for select all checkbox using property "ariaLabelForSelectAllCheckbox"

**Comment by pagangil** [comment]
Any updates on this?
